src/c/project-card-edit.js

```diff
@@ -81,6 +82,23 @@ const projectCardEdit = {
             m('form.w-form', { onsubmit: ctrl.onSubmit }, [
                 m('.w-section.section', [
                     m('.w-container', [
+                        (vm.currentProject().mode === 'sub' ?
```

Comment:
I added a check for project mode on project-vm. We can extend it to maybe check the currentProject() if any project is given as a parameter and if a currentProject is set.

Reply:
I'll merge it anyways.